shipwright/build.py

```diff
@@ -50,9 +51,27 @@ def build(client, git_rev, container):
         'rev': git_rev
     }
 
-    def process_event_(evt):
-        evt_parsed = json_loads(evt)
-        return merge(merge_config)(evt_parsed)
+    # docker-py has an issue where it doesn't handle chunked responses from
```

Comment:
I can, but it would need to be merged for me to be able to carry on working on this PR as its a build failure for me.